### ISSUE TITLE
[accname] Two necessary steps for using aria-label values in accessible-name computation are missing from the spec

### ISSUE THREAD

**Issue by sideshowbarker** [w3c/aria#2388]
https://w3c.github.io/accname/#comp_labelledby

I have found that:

1. The _“div group explicitly labelledby self and heading”_ subtest at https://wpt.fyi/results/accname/name/comp_labelledby.html won’t pass unless when performing the “For each IDREF” substep of the “LabelledBy” substep in the accessible-name computation algorithm, I check for any `aria-label` attribute and use its value if found.
2.  The _“button's hidden referenced name (visibility:hidden) with hidden aria-labelledby traversal falls back to aria-label”_ subtest at https://wpt.fyi/results/accname/name/comp_labelledby.html won’t pass unless when performing the “Return the accumulated text” substep of the “LabelledBy” substep in the accessible-name computation algorithm, I check to determine of the `total_accumulated_text` value is empty, and if so, then check for any `aria-label` attribute and use its value if found.

### Link to the version of the specification or documentation you were looking at at.

https://w3c.github.io/accname/#comp_labelledby

Does the issue exists in [the editors draft](https://w3c.github.io/aria/) (the editors draft is the most recent draft of the specification)? Yes
